gradle-tests/build.gradle

```diff
@@ -1,6 +1,6 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 plugins {
-    id 'com.android.library' version '8.2.0' apply false
+    id 'com.android.library' version '8.3.0-alpha17' apply false
```

Comment:
Do you have a ref to the feature/fix that this pulls in?

Reply:
Here is the list of patches that are included in 8.3.0-alpha17
- https://cs.android.com/android-studio/platform/tools/base/+/719b3d9d01b419786f3411dd8f6b4fc4f73b92f5
- https://cs.android.com/android-studio/platform/tools/base/+/ed331de5664d73f375561522c3b182c81de475b3
- https://cs.android.com/android-studio/platform/tools/base/+/97a827fedb3026bf07c35b8fab5c3da0180e7320
- https://cs.android.com/android-studio/platform/tools/base/+/d05e43ef1d4bbedc36100036b9d89acfb0337fa3

Reply:
Thanks! (middle two not working)